ui/public/darkmode.css

```diff
@@ -0,0 +1,23 @@
+
+body, nav {
+  color: rgb(255, 253, 248);
+  background: rgb(19, 19, 20);
+}
+
+a, .collapsible label, .svelte-gwqplq {
```

Comment:
We shouldn't rely on css classes generated by the framework

Reply:
I don't know, where this particular one is coming from, but if necessary, we could move it from its respective component to the global css.

Reply:
I am removing this in the next commit. This was an artifact from a different button implementation that would stay white without this when in darkmode but is no longer necessary.